types/tizentv__webide-common-tizentv/index.d.ts

```diff
@@ -0,0 +1,99 @@
+// Type definitions for @tizentv/webide-common-tizentv 1.0
+// Project: https://github.com/Samsung/webIDE-common-tizentv
+// Definitions by: Bart van den Ende <https://github.com/bartvandenende-wm>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.8
```

Comment:
```suggestion
```
I think this is not required, why this version limit? (default version should work just fine I believe)